crowbar_framework/db/migrate/20160215120522_add_migration_level_to_backups.rb

```diff
@@ -0,0 +1,5 @@
+class AddMigrationLevelToBackups < ActiveRecord::Migration
+  def change
+    add_column :backups, :migration_level, :integer, limit: 8
```

Comment:
we could use `default: 20151222144602` like for the backup upload in https://github.com/crowbar/crowbar-core/pull/343/files#diff-408ac208d889893305de65518aa046a2R180 ?